src/style.css

```diff
@@ -0,0 +1,95 @@
+/* 20220620 properties "background" structured for better understanding */
+/* 20220622 selector "input" changed to class "textSearch" */
+/* 20220622 added color to class "textSearch" and attribute "placeholder" */
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.searchBar {
+  margin: 20px 10px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61,78,97,0.1);
+}
+
+.searchBar[data-qa="big"] {
+  height: 70px;
+}
+
+.searchBar[data-qa="small"] {
+  height: 42px;
+}
```

Comment:
use classes instead of data attributes.

first element would have one class: searchBar
second element would have two classes: searchBar and searchBar--small